evm/vm/forks/byzantium/__init__.py

```diff
@@ -16,20 +13,15 @@
 from .state import ByzantiumState
 
 
-def make_byzantium_receipt(transaction, computation, state):
-    old_receipt = make_frontier_receipt(transaction, computation, state)
+def make_byzantium_receipt(base_header, transaction, computation, state):
+    frontier_receipt = make_frontier_receipt(base_header, transaction, computation, state)
 
     if computation.is_error:
-        state_root = EIP658_TRANSACTION_STATUS_CODE_FAILURE
+        success_code = EIP658_TRANSACTION_STATUS_CODE_FAILURE
     else:
-        state_root = EIP658_TRANSACTION_STATUS_CODE_SUCCESS
+        success_code = EIP658_TRANSACTION_STATUS_CODE_SUCCESS
```

Comment:
I'm wrapping for the day, but on another review, this should probably be `status_code` instead of `success_code`